lib/ace/css/editor.css

```diff
@@ -250,6 +250,17 @@
     box-sizing: border-box;
 }
 
+.ace_marker-layer .ace_easymotion-char {
```

Comment:
This probably needs a variant for .ace_dark too to style light and dark themes differently.